### ISSUE TITLE
[BUG][code-analyzer] <YOUR_TITLE_HERE>

### ISSUE THREAD

**Issue by DanielGalloKaseya** [forcedotcom/code-analyzer#1958]
### Have you tried to resolve this issue yourself first?

- [x] I confirm I have gone through the above steps and still have an issue to report.

### Bug Description

 flow:UnexpectedEngineError   null:undefined:undefined  no such file or directory,open 'C:\Users\DANIEL~1.GAL\AppData\Local\Temp\code-analyzer-Tea130\run-2025_12_19_13_39_40_449\flow\flowScannerResultsFile.json'

### Output / Logs

```shell
1   1 (Critical)   flow:UnexpectedEngineError   null:undefined:undefined   The engine with name 'flow' threw an unexpected error: Error: ENOENT: no such file or directory, open 'C:\Users\DANIEL~1.GAL\AppData\Local\Temp\code-analyzer-Tea130\run-2025_12_19_13_39_40_449\flow\flowScannerResultsFile.json'
                                                                             at async open (node:internal/fs/promises:642:25)
                                                                             at async Object.readFile (node:internal/fs/promises:1279:14)
                                                                             at async RunTimeFlowScannerCommandWrapper.runFlowScannerRules (C:\Users\daniel.gallo\AppData\Local\sf\node_modules\@salesforce\plugin-code-analyzer\node_modules\@salesforce\code-analyzer-flow-engine\dist\python\FlowScannerCommandWrapper.js:52:36)
                                                                             at async FlowScannerEngine.runRules (C:\Users\daniel.gallo\AppData\Local\sf\node_modules\@salesforce\plugin-code-analyzer\node_modules\@salesforce\code-analyzer-flow-engine\dist\engine.js:104:34)
                                                                             at async CodeAnalyzer.runEngineAndValidateResults (C:\Users\daniel.gallo\AppData\Local\sf\node_modules\@salesforce\plugin-code-analyzer\node_modules\@salesforce\code-analyzer-core\dist\code-analyzer.js:389:35)
                                                                             at async C:\Users\daniel.gallo\AppData\Local\sf\node_modules\@salesforce\plugin-code-analyzer\node_modules\@salesforce\code-analyzer-core\dist\code-analyzer.js:276:29
                                                                             at async Promise.all (index 0)
                                                                             at async CodeAnalyzer.run (C:\Users\daniel.gallo\AppData\Local\sf\node_modules\@salesforce\plugin-code-analyzer\node_modules\@salesforce\code-analyzer-core\dist\code-analyzer.js:283:38)
                                                                             at async RunAction.execute (file:///C:/Users/daniel.gallo/AppData/Local/sf/node_modules/@salesforce/plugin-code-analyzer/lib/lib/actions/RunAction.js:39:25)
                                                                             at async RunCommand.run (file:///C:/Users/daniel.gallo/AppData/Local/sf/node_modules/@salesforce/plugin-code-analyzer/lib/commands/code-analyzer/run.js:86:9)



=== Summary

Found 1 violation(s) across 0 file(s):
    1 Critical severity violation(s) found
```

### Steps To Reproduce

sf code-analyzer run --rule-selector flow --target C:\xxxxx\unpackaged\main\default\flows\Test_Security_Flow.flow-meta.xml --output-file C:\xxxx\Test_Security_Flow.flow-meta.xml.pmd.html --view table

### Expected Behavior

THE CORRECT REPORT WITHOUT DE ISSUE FROM PLUGIN ENGIE FLOW

### Operating System

windows 11

### Salesforce CLI Version

@salesforce/cli/2.116.6 win32-x64 node-v24.11.1

### Code Analyzer Plugin (code-analyzer) Version

flow 1.0.5

### Node Version

24.11.1 (Currently using 64-bit executable)

### Java Version

openjdk 25.0.1 2025-10-21 LTS OpenJDK Runtime Environment Temurin-25.0.1+8 (build 25.0.1+8-LTS) OpenJDK 64-Bit Server VM Temurin-25.0.1+8 (build 25.0.1+8-LTS, mixed mode, sharing)

### Python Version

Python 3.14.2

### Additional Context (Screenshots, Files, etc)

<?xml version="1.0" encoding="UTF-8"?>
<Flow xmlns="http://soap.sforce.com/2006/04/metadata">
    <apiVersion>59.0</apiVersion>
    <decisions>
        <name>Check_Permission</name>
        <label>Check Permission</label>
        <locationX>200</locationX>
        <locationY>100</locationY>
        <defaultConnectorLabel>Default Outcome</defaultConnectorLabel>
        <rules>
            <name>Admin_Only</name>
            <conditionLogic>and</conditionLogic>
            <connector>
                <isGoTo>true</isGoTo>
                <targetReference>Allowed</targetReference>
            </connector>
            <label>Admin Only</label>
            <ruleConnections>
                <sourceId>Check_Permission</sourceId>
                <targetId>Admin_Only</targetId>
            </ruleConnections>
        </rules>
    </decisions>
    <label>Test Security Flow</label>
    <processMetadataValues>
        <name>BuilderType</name>
        <value>
            <stringValue>Flow</stringValue>
        </value>
    </processMetadataValues>
    <processType>Flow</processType>
</Flow>

### Urgency

Hight
